src/main/java/com/hackclub/hccore/slack/SlackBot.java

```diff
@@ -51,81 +69,125 @@ public SlackBot(HCCorePlugin plugin) throws Exception {
         return ctx.ack();
       }
       String userId = event.getUser();
-      UsersProfileGetResponse result = ctx.client().usersProfileGet(r ->
-          r
-              .token(ctx.getBotToken())
-              .user(userId)
-      );
+      UsersProfileGetResponse result = ctx.client()
+          .usersProfileGet(r -> r.token(ctx.getBotToken()).user(userId));
       String displayName = result.getProfile().getDisplayName();
 
-      TextComponent prefixComponent = Component.text("[Slack] ")
-          .color(NamedTextColor.BLUE);
+      TextComponent prefixComponent = Component.text("[Slack] ").color(NamedTextColor.BLUE);
 
-      TextComponent nameComponent = Component.text(displayName)
-          .color(NamedTextColor.WHITE)
+      TextComponent nameComponent = Component.text(displayName).color(NamedTextColor.WHITE)
           .hoverEvent(Component.text(result.getProfile().getRealName()));
 
-      TextComponent arrowComponent = Component.text(" » ")
-          .color(NamedTextColor.GOLD);
+      TextComponent arrowComponent = Component.text(" » ").color(NamedTextColor.GOLD);
 
       TextComponent playerChatComponent = Component.text(
           ChatColor.translateAlternateColorCodes('&', text)).color(NamedTextColor.GRAY);
 
-      plugin.getServer().broadcast(prefixComponent
-          .append(nameComponent)
-          .append(arrowComponent)
-          .append(playerChatComponent)
-      );
+      plugin.getServer().broadcast(
+          prefixComponent.append(nameComponent).append(arrowComponent).append(playerChatComponent));
 
       return ctx.ack();
     });
 
     app.event(MessageBotEvent.class, (payload, ctx) -> ctx.ack());
 
+    app.event(MessageDeletedEvent.class, (payload, ctx) -> ctx.ack());
+
+    app.event(MessageChannelJoinEvent.class, (payload, ctx) -> ctx.ack());
+
+    CommandDispatcher<SlashCommandRequest> dispatcher = new CommandDispatcher<>();
+    // TODO change to get base command from config.yml
+    dispatcher.register(
+        LiteralArgumentBuilder.<SlashCommandRequest>literal("/%s".formatted(commandBase))
+            .then(
+                LiteralArgumentBuilder.<SlashCommandRequest>literal("players").executes(context -> {
+                  StringBuilder message = new StringBuilder(
+                      "Players online (%d/%d)\n\n".formatted(
```

Comment:
I think that there should be a case added to this where if there's nobody online just say "There are no players online".

Reply:
I also think that the "Players online" line of the message should be bolded

Reply:
done